src/components/Status/Status.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC, useMemo } from 'react';
+
+import { StatusProps } from './Status.props';
+import { stylesBuilder } from './stylesBuilder';
+
+import { tet } from '@/tetrisly';
+
+export const Status: FC<StatusProps> = ({
+  appearance,
+  emphasis,
+  label,
+  custom,
+  ...restProps
+}) => {
+  const styles = useMemo(
+    () => stylesBuilder(appearance, custom, emphasis),
+    [custom, emphasis, appearance],
+  );
+
+  return (
+    <tet.div {...styles.container} data-testid="status" {...restProps}>
+      <tet.span {...styles.dot} data-testid="status-dot" />
+      <tet.span>{label}</tet.span>
```

Comment:
technically there are styles in this label container in Figma, but you've put them in the parent container and all of them are being passed down (like color and typography), imo looks good. Unless @adrian-potepa thinks otherwise

Reply:
Agree, less html is always better solution.

Reply:
ok, so I assume a conversation is resolved :)

Reply:
```suggestion
     {label}
```